http4s/src/main/twirl/com/iheart/thomas/abtest/admin/index.scala.html

```diff
@@ -162,7 +162,7 @@ <h5 class="mt-3 mb-3">A/B tests</h5>
                         @if(env.currentUser.managing(features).nonEmpty) {
                         <div class="form-row mb-3">
                             <label class="form-label" for="featuresSelect">Or for an existing feature</label>
-                            <select id="featuresSelect" name="feature" class="form-control form-filter">
+                            <select id="featuresSelect" name="feature" class="form-control form-filter selectSearch">
                                 @for(feature <- env.currentUser.managing(features).sortBy(_.name)) {
```

Comment:
nitpick, you don't need a new class here. can simply use the id "featuresSelect"

Reply:
there is another dropdown [menu](https://github.com/iheartradio/thomas/pull/803/files#diff-c87eeecabe196dc85bce4a9713896032a9d4847053468e7389c5853380654376R110) i made searchable. so i added a new class `selectSearch` to both dropdowns. if i use featuresSelect, then i will do $('#featuresSelect').select2({}) and $('#testsFeature').select2({}). is that ok with you?